### PR TITLE
chore: add volunteer KrishnaTheCoder756

### PR DESCRIPTION
**Pull request by KrishnaTheCoder756**
chore: add volunteer KrishnaTheCoder756

### CODE REVIEW

**Review by KrishnaTheCoder756 — COMMENTED** [review]
chore: Change the profile picture for KrishnaTheCoder756

File: data/users/KrishnaTheCoder756.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
+{
+    "name": "Krishna Sarathi Ghosh",
+    "bio": "Python Developer | Learning Full Stack Web Development",
```
Comment (codewithvoid) [inline data/users/KrishnaTheCoder756.json:3]: Can you just mention Python Web Developer. That will be short and clear.

Comment (KrishnaTheCoder756) [reply data/users/KrishnaTheCoder756.json:3]: Ok

**Comment by codewithvoid** [comment]
Thanks for making the changes @KrishnaTheCoder756. Welcome to the community!

**Pull request merged**